build/preset.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Generate a Babel preset that outputs ES modules when the env is `es`, and
+ * CommonJS modules otherwise.
+ */
+
+const es = (process.env.BABEL_ENV || process.env.NODE_ENV) === 'es';
+
+module.exports = {
+  presets: [
+    ['es2015', {
+      modules: es ? false : 'commonjs',
```

Comment:
We'd have to "duplicate" the file still with that. The `env` option is also being phased out in Babel v7, instead favouring a JS file :p I'll push a different approach tho, that may be clearer.